ui/src/components/shared/ExemplaryProject.js

```diff
@@ -133,9 +133,11 @@ function ExemplaryProject({ project }) {
         </div>
       </div>
       {/* Modal with expanded information */}
+      // Around line 134, change the Modal to:
```

Comment:
Wrong format used for HTML comment, resulting in this text showing up under each exemplary project.
```suggestion
      {/* Around line 134, change the Modal to: */}
```